Prism/pom.xml

```diff
@@ -34,7 +34,7 @@
         <dependency>
             <groupId>net.kyori</groupId>
             <artifactId>adventure-api</artifactId>
-            <version>4.1.1</version>
+            <version>4.7.0</version>
```

Comment:
@darbyjack Just curious, is this version bump related to the fix? I assume not, but it wasn't mentioned in the PR description. It's not a blocker, but I want to be sure unrelated changes aren't being pulled in :slightly_smiling_face:

Reply:
Also sorry we haven't been responsive lately, life getting in the way and all... I'm happy to approve this, just wanted to clarify regarding the `adventure` changes first

Reply:
I want to say there was some error at some point related to it, yes. I think it was with the LegacySerializer / Deserializer. I don't have logs all the way from back then anymore but I made the change due to some error when I was trying to fix the MySQL.

Reply:
All good, I think it's safe to merge this now :+1: Nice work, thanks!! :clap: